src/applications/caregivers/tests/e2e/1010cg.cypress.spec.js

```diff
@@ -3,102 +3,62 @@ import testForm from 'platform/testing/e2e/cypress/support/form-tester';
-  'I certify that I am at least 18 years of age.',
-  'I certify that either: (1) I am a member of the Veteran\u2019s family (including a parent, spouse, a son or daughter, a step-family member, or an extended family member) OR (2) I am not a member of the Veteran\u2019s family, and I reside with the Veteran full-time or will do so upon designation as the Veteran\u2019s Secondary Family Caregiver.',
-  'I agree to perform personal care services as the Secondary Family Caregiver for the Veteran named on this application.',
-  'I understand that the Veteran or the Veteran\u2019s surrogate may request my discharge from the Program of Comprehensive Assistance for Family Caregivers (PCAFC) at any time and that my designation as a Secondary Family Caregiver may be revoked or I may be discharged from PCAFC by the Secretary of Veterans Affairs (or designee) as set forth in 38 CFR 71.45.',
-  'I understand that participation in the PCAFC does not create an employment relationship between me and the Department of Veterans Affairs.',
-];
-
-const checkContent = (partyLabel, content, mockContent) => {
-  content.forEach((contentItem, idx) => {
-    cy.get(`[data-testid="${partyLabel}"]`)
-      .contains(contentItem, { matchCase: true })
-      .should(signatureParagraph =>
-        expect(signatureParagraph[0].innerText).to.eq(mockContent[idx]),
-      );
-  });
-};
-
-const signAsParty = (partyLabel, signature) => {
-  cy.findByTestId(partyLabel)
-    .find('.signature-input')
-    .shadow()
-    .find('input')
-    .first()
-    .type(signature);
-
-  cy.findByTestId(partyLabel)
-    .find('.signature-checkbox')
-    .shadow()
-    .find('label')
-    .click();
-};
-
-const testSecondaryTwo = createTestConfig(
+const testConfig = createTestConfig(
   {
     dataPrefix: 'data',
     dataSets: [
       'requiredOnly',
       'secondaryOneOnly',
-      'oneSecondaryCaregivers',
+      'oneSecondaryCaregiver',
       'twoSecondaryCaregivers',
-      // 'signAsRepresentativeNoRep',
       'signAsRepresentativeNo',
       'signAsRepresentativeYes',
     ],
     fixtures: {
       data: path.join(__dirname, 'fixtures', 'data'),
       mocks: path.join(__dirname, 'fixtures', 'mocks'),
     },
-
     setupPerTest: () => {
       cy.intercept('GET', '/v0/feature_toggles?*', featureToggles);
       cy.intercept('POST', 'v0/form1010cg/attachments', mockUpload);
       cy.intercept('GET', '/v1/facilities/va?*', mockFacilities).as(
         'getFacilities',
       );
+      cy.intercept('POST', '/v0/caregivers_assistance_claims', mockSubmission);
     },
     pageHooks: {
       introduction: ({ afterHook }) => {
         afterHook(() => {
-          // Hit the start button
-          cy.findAllByText(/start/i, { selector: 'a' })
+          cy.get('[href="#start"]')
             .first()
             .click();
         });
       },
-      'select-facility': ({ afterHook }) => {
+      'veteran-information/home-address': ({ afterHook }) => {
```

Comment:
All address pages need to be manually declared to work around an issue where state options are not populated on render due to the `updateSchema` method that needs to run once country is populated.